managed/ui/src/components/common/forms/RegisterForm/RegisterForm.js

```diff
@@ -23,14 +29,27 @@ class RegisterForm extends Component {
   }
 
   submitRegister = (formValues) => {
-    const { registerCustomer } = this.props;
+    const config = {
+      type: 'PASSWORD_POLICY',
+      name: 'password policy',
+      data: {
+        minLength: 8,
+        minUppercase: 1,
+        minLowercase: 1,
+        minDigits: 1,
+        minSpecialCharacters: 1
```

Comment:
We should be storing the password configuration that the backend returns and not hard-coding it here.

Reply:
@yb-andrew : Just a question, I agree we should not hardcode the config data here, but we do need to pass the config data that needs to be stored in backend. Do I need to pass the above config data in the `[create](https://github.com/yugabyte/yugabyte-db/blob/master/managed/src/main/java/com/yugabyte/yw/controllers/CustomerConfigController.java#L46)` function in `CustomerConfigController.java` when the formData type is `PASSWORD_CONFIG` ?

Reply:
Background of why I add the config above:

So, while customer is not yet registered - it can't have custom password policy configured - so default one is used at this point.
We should use customer-specific policy only when customer is already created.
So we can create customer policy via the following API once customer is registered:

POST /api/v1/customers/<customer_uuid>/configs

with the body like that:

{
  "name": "password policy",
  "type": "PASSWORD_POLICY",
  "data": {
     "minLength": 8,
     "minUppercase": 1,
     "minLowercase": 1,
     "minDigits": 1,
     "minSpecialCharacters": 1
  }
}

This is what I have done, please let me know

Reply:
@rajmaddy89 When the customer is not registered, there is no customerUUID, but the password policy can be retrieved from the server's configuration (kind of like the environment variables). This code should look similar to the original code in `PasswordPolicyService.java` file. Here is pseudo-code
```
PasswordPolicyFormData effectivePolicy;
if (configuredPolicy == null) { // If customer is new, there will not be a configured policy yet
      effectivePolicy = new PasswordPolicyFormData();
      effectivePolicy.setMinLength(config.getInt(DEFAULT_MIN_LENGTH_PARAM));
      effectivePolicy.setMinUppercase(config.getInt(DEFAULT_MIN_UPPERCASE_PARAM));
      effectivePolicy.setMinLowercase(config.getInt(DEFAULT_MIN_LOWERCASE_PARAM));
      effectivePolicy.setMinDigits(config.getInt(DEFAULT_MIN_DIGITS_PARAM));
      effectivePolicy.setMinSpecialCharacters(config.getInt(DEFAULT_MIN_SPECIAL_CHAR_PARAM));
}
```